src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
```

Comment:
- [ ] As mentioned by the previous reviewer kindly make sure When the user clicks on the "Back" button (<), the user navigates to the home page.